views/suggestions.html

```diff
@@ -80,7 +79,7 @@ <h1>Share It</h1>
 
 		<h1 class="center" id="heading">Suggest something about Repl.it!</h1>
 		<div id="form" class="center">
-			<form>
+			<form action="/api/suggestions" method="POST">
```

Comment:
I would say this is bad practice, but I have no idea how actions work so I have a workaround using simple js

Reply:
Why is this bad practice. I am not done learning JS yet...Or CSS

Reply:
It's not, but I have no idea how `action` works and the docs don't explain very well. I think it performs a `PUT` request against the uri provided, but I have no idea.